### PR TITLE
add subtitle for ZAP Chat Video 09-13

### PR DESCRIPTION
**Pull request by 8linkz**
Video url:
Video 09: https://www.youtube.com/watch?v=4phnMy9iCPY
Video 10: https://www.youtube.com/watch?v=WivoyVerBCo
Video 11: https://www.youtube.com/watch?v=xuP00Ri460k
Video 12: https://www.youtube.com/watch?v=hcftgjz_Vgc
Video 13: https://www.youtube.com/watch?v=VPVuxftv99A

Especially with the .srt file of video 12 I had to do a lot of manual work due to missing punctuation and hallucinations. You might have to check this again.

### CODE REVIEW

**Comment by kingthorin** [comment]
Could you skip all the commits from the previous PR?

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

**Comment by 8linkz** [comment]
> Could you skip all the commits from the previous PR?

I have to admit, I searched the internet, but somehow I couldn't find the point that would help me - I would have to close the PR, make a new branch and then reopen it, right?

**Comment by kingthorin** [comment]
No you don't need a new PR. You'd just have to do an interactive rebase then force push.

Here's some info that should help:

- https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
- https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History
- https://www.instructables.com/id/How-to-Rebase-Interactive-With-Eclipse-eGit/
- https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase

**Comment by 8linkz** [comment]
> No you don't need a new PR. You'd just have to do an interactive rebase then force push.
> 
> Here's some info that should help:
> 
>     * https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
> 
>     * https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History
> 
>     * https://www.instructables.com/id/How-to-Rebase-Interactive-With-Eclipse-eGit/
> 
>     * https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase

Thanks I hope that fits at least as far as the commits are concerned. I can't squash the first one because it always wants a "first one" (error: cannot 'squash' without a previous commit).

I haven't understood the signoff yet, I'll have to look again.

**Comment by kingthorin** [comment]
Are you using git on the command line?

**Comment by 8linkz** [comment]
> Are you using git on the command line?

VSCode -> Terminal
and before that I did it via github without a terminal...

Edit:
I have merged one such commit, which was not signed. I think the problem now is that it was previously signed via Github and now I have stored meaningful contact details...I don't think I can resolve that now.

**Comment by thc202** [comment]
Fetch from upstream and rebase (it's better to work on other branches than main, it might cause problems when force pushing).

**Comment by 8linkz** [comment]
> Fetch from upstream and rebase (it's better to work on other branches than main, it might cause problems when force pushing).

HaHa - so done - I have now understood the principle behind it and how I can resolve it.

Now everything can be checked here.

**Comment by thc202** [comment]
Seems your branch is still 1 commit behind so not rebased (the older files should not be shown as new in this PR).

**Comment by 8linkz** [comment]
> Seems your branch is still 1 commit behind so not rebased (the older files should not be shown as new in this PR).

grafik
if i try to rebase the first commit (squash) 

I think this will unfortunately be a remnant because I stupidly used the main branch and not a separate one just for the PR.

**Comment by kingthorin** [comment]
Skip it instead of squashing. 

Skip
Pick 
Squash
Squash
Squash

(Or even fixup instead of squash)

**Comment by 8linkz** [comment]
> Seems your branch is still 1 commit behind so not rebased (the older files should not be shown as new in this PR).

I think I found my mistake here - please check it again when you get a chance. Thank you very much.

**Comment by kingthorin** [comment]
The commits are as expected, as far as I can tell. I haven't reviewed the text yet but will try to in the next few days.

**Comment by thc202** [comment]
This was not yet rebased though (not that it needs in this case, with the other commit removed it is no longer adding old files), you would have to rebase on e.g. `upstream/main`.

**Comment by kingthorin** [comment]
git fetch upstream 
git rebase upstream/main
git push --force

**Comment by 8linkz** [comment]
@thc202 
The 5 commits ahead are probably from the pull request - this also has 5. Since I did this from main to main and without a separate branch, this will be the cause here. 

grafik

**Comment by thc202** [comment]
Yes, that's correct.

File: subtitles/ZAP Chat/ZAP_Chat_12_Automation_Framework_Part_6_-_Delays_and_Active_Scan_english.srt [hunk 1]
```diff
@@ -0,0 +1,1608 @@
...
+00:00:14,640 --> 00:00:23,220
+Great, so last week we covered spiders, both the traditional spider and the AJAX spider.
+
+4
+00:00:24,200 --> 00:00:29,980
+So hopefully today we're going to get on to ActiveScan, but before we do that, we've got a couple of little jobs.
+
+5
+00:00:30,720 --> 00:00:36,040
+Now, one of them I already mentioned, which was the PassiveScan weight.
```
Comment (psiinon) [inline subtitles/ZAP Chat/ZAP_Chat_12_Automation_Framework_Part_6_-_Delays_and_Active_Scan_english.srt]: weight -> wait :)

Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_12_Automation_Framework_Part_6_-_Delays_and_Active_Scan_english.srt]: done

**Review by psiinon — APPROVED** [review]

File: subtitles/ZAP Chat/ZAP_Chat_09_Automation_Framework_Part_3_english.srt [hunk 2]
```diff
@@ -0,0 +1,1212 @@
...
+00:08:39,560 --> 00:08:46,440
+So we've got the test header there and in the post, we don't have the test header because we didn't add it.
+
+129
+00:08:46,960 --> 00:08:48,560
+And we've got the data there.
+
+130
+00:08:49,640 --> 00:08:52,020
+Well, one header that has been added is the content left.
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_09_Automation_Framework_Part_3_english.srt]: left → length

File: subtitles/ZAP Chat/ZAP_Chat_10_Automation_Framework_Part_4_-_Spidering_english.srt [hunk 3]
```diff
@@ -0,0 +1,1671 @@
...
+00:00:00,000 --> 00:00:12,000
+Hello and welcome to another episode of ZAP Chat with me, Simon Bennetts and Yiannis.
+
+2
+00:00:12,000 --> 00:00:16,000
+Hello! Spidering is the topic of the day today, right Simon?
+
+3
+00:00:16,000 --> 00:00:25,000
+That's right. I think in the last one I said we were going to do API imports, but we've had a chat and we think spides are probably better.
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_10_Automation_Framework_Part_4_-_Spidering_english.srt]: spides → spiders

File: subtitles/ZAP Chat/ZAP_Chat_09_Automation_Framework_Part_3_english.srt (hunk 2 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_09_Automation_Framework_Part_3_english.srt]: done

File: subtitles/ZAP Chat/ZAP_Chat_10_Automation_Framework_Part_4_-_Spidering_english.srt (hunk 3 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_10_Automation_Framework_Part_4_-_Spidering_english.srt]: done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**